test/cpp/jit/test_upgrader_utils.cpp

```diff
@@ -58,5 +60,31 @@ TEST(UpgraderUtils, FindIfOpIsCurrent) {
   test_only_remove_entry("foo");
 }
 
+TEST(UpgraderUtils, CanLoadHistoricOp) {
+  std::vector<UpgraderEntry> dummy_entry = {
+      {4, "foo__0_3", "foo.bar()"},
+      {8, "foo__4_7", "foo.foo()"},
+  };
+
+  std::vector<std::string> schemas = {"foo.bar()", "foo.foo()"};
+
+  // symbol based look up
+  test_only_add_entry("foo.first", dummy_entry[0]);
```

Comment:
I would suggest using a more exotic name like "an_op_that_does_not_exit" to avoid accidental collisons.

Reply:
done